superscore/widgets/filter_page.py

```diff
@@ -0,0 +1,54 @@
+'''
+Filter dialog page .py file
+'''
+from qtpy import QtWidgets
+
+from superscore.widgets.core import Display
+
+
+class FilterMenu(Display, QtWidgets.QWidget):
+    filename = "filterScreen.ui"
+
+    # filter snapshot horizaontal layout
+    clearFilterButton: QtWidgets.QPushButton
+    filterSnapshotLabel: QtWidgets.QLabel
+
+    # Filter options vert layout
+    collResetButton: QtWidgets.QPushButton
+    collectionsLable: QtWidgets.QLabel
+    collectionsTreeWidget: QtWidgets.QTreeWidget
+
+    metaPvLabel: QtWidgets.QLabel
+    metaPvResetButton: QtWidgets.QPushButton
+    metaPvTableView: QtWidgets.QTableView
+
+    tagLabel: QtWidgets.QLabel
+    tagListWidget: QtWidgets.QListWidget
+    tagResetButton: QtWidgets.QPushButton
+
+    applyFiltersButton: QtWidgets.QPushButton
+
+    def saveData(self):
+        pass
+
+    def getCollections(self):
+        pass
+
+    def getMetaPV(self):
+        pass
+
+    def getTags(self):
+        pass
+
+    def fillDataCollections(self):
+        pass
+
+    def fillDataMetaPV(self):
+        pass
+
+    def fillDataTags(self):
+        pass
+
+    def __init__(self, *args, **kwargs):
+        super(FilterMenu, self).__init__(*args, **kwargs)
```

Comment:
```suggestion
        super().__init__(*args, **kwargs)
```
You don't need to specify the class here anymore.  I think this changed in the python 2->3 transition.  This applies for all the other `super()` calls as well

Reply:
fixed, removed on all files